rbacutils/rbacprocessor.go

```diff
@@ -65,12 +63,15 @@ func InitSAID2WLIDmap(k8sAPI *k8sinterface.KubernetesApi, clusterName string) (m
 	}
 	allworkloads, err := ListAllWorkloads(k8sAPI)
 	if err != nil {
-		logger.L().Warning(err.Error())
 		return saID2WLIDmap, nil
 	}
 	for _, wl := range allworkloads {
-		if ref, err := wl.GetOwnerReferences(); len(ref) == 0 && err == nil {
-			saID := fmt.Sprintf("/%s/%s/%s/%s", serviceaccountversion, wl.GetNamespace(), serviceaccountkind, wl.GetServiceAccountName())
+		if !k8sinterface.WorkloadHasParent(wl) {
+			connectedSA := wl.GetServiceAccountName()
+			if connectedSA == "" {
+				connectedSA = "default"
+			}
+			saID := "/" + strings.Join([]string{serviceaccountversion, wl.GetNamespace(), serviceaccountkind, connectedSA}, "/")
```

Comment:
Do we parse the ID anywhere? If not, we don't need to add the "/" prefix.

Reply:
In the UI, I don't want to break anything there